docs/r2/hyperparameter_tuning_with_hparams.ipynb

```diff
@@ -0,0 +1,599 @@
+{
```

Comment:
<p>I feel like a motion gif or short video would be ideal here.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Adding an image (or GIF or video) would be a short-term solution until the docs site is updated. Something with motion might make the experience even more confusing when running within Colab (there will be a live TensorBoard and then something moving under it). I'm tempted to leave this as it is for now. WDYT?

Reply:
<p>Yeah, simplicity is better.</p>